### ISSUE TITLE
OpenVPN Connect Client 3.download.recipe doesn't work without Munki recipe

### ISSUE THREAD

**Issue by nstrauss** [autopkg/dataJAR-recipes#307]
https://github.com/autopkg/dataJAR-recipes/blob/master/OpenVPN%20Connect%20Client/OpenVPN%20Connect%20Client%203.download.recipe

Since there's no `ARCH` input variable in the download recipe, it doesn't work without the accompanying Munki recipe. Add...

```XML
<dict>
    <key>ARCH</key>
    <string>arm64</string>
    <key>NAME</key>
    <string>OpenVPNConnectClient3</string>
</dict>
```

EDIT: There's also a space on this line 🫠
https://github.com/autopkg/dataJAR-recipes/blob/a78f11c18bab8708bccf716b303bcecc74abc973/OpenVPN%20Connect%20Client/OpenVPN%20Connect%20Client%203.munki.recipe#L19

**Comment by paul-cossey** [comment]
Thanks for raising this, @nstrauss 

Could you give https://github.com/autopkg/dataJAR-recipes/compare/fbefb8e91fde...c475a8aa23d9 at try please?

**Comment by nstrauss** [comment]
@paul-cossey I see you've already merged c475a8aa23d95ae24c41b47808c175fbc2b3c9c2 which I've tested as working. Thank you!
